dashboards/bigquery/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+id: resource_list/bigquery_dataset
```

Comment:
This file should match the format used in other directories in this repository, e.g. https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/google-anthos/metadata.yaml

This file needs to support multiple entries, so there should be a top level `sample_dashboards` key, with a list of dashboards nested underneath.

Reply:
refactored file.